vision/classification_and_detection/README.md

```diff
@@ -72,7 +72,7 @@ python tools/accuracy-coco.py --mlperf-accuracy-file mlperf_log_accuracy.json --
 | coco (validation) | http://images.cocodataset.org/zips/val2017.zip |
 | coco (annotations) | http://images.cocodataset.org/annotations/annotations_trainval2017.zip |
 | openimages | We provide a [script](tools/openimages_mlperf.sh) to download the openimages mlperf validation set. You can download the dataset by going into the tools folder and running `./openimages_mlperf -d <DOWNLOAD_PATH>` |
-| openimages (calibration) | We also provide a [script](tools/openimages_calibration_mlperf.sh) to download the openimages mlperf validation set. You can download the dataset by going into the tools folder and running `./openimages_calibration_mlperf -d <DOWNLOAD_PATH>`. This requires you to have [calibration list](../../calibration/openimages/openimages_cal_images_list.txt) |
+| openimages (calibration) | We also provide a [script](tools/openimages_calibration_mlperf.sh) to download the openimages mlperf validation set. You can download the dataset by going into the tools folder and running `./openimages_calibration_mlperf -d <DOWNLOAD_PATH> -m <MAX_IMAGES>`. This requires you to have [calibration list](../../calibration/openimages/openimages_cal_images_list.txt) |
```

Comment:
It should be "s/validation/calibration" set here right?

Reply:
I only put the flag in the validation script, since the calibration only downloads 500 images. But you are right that I messed up the readme. This flag shouldn't be for `./openimages_calibration_mlperf.sh` (line 75), instead it should be for `./openimages_mlperf.sh` (line 74). I'll push that fix